flask_multipass_cern.py

```diff
@@ -267,6 +274,7 @@ def search_identities_ex(self, criteria, exact=False, limit=None):
 
     def get_identity_groups(self, identifier):
         with self._get_api_session() as api_session:
+            api_session.mount(self.authz_api_base, retry_config)
```

Comment:
should this be done here or in `_get_api_session`?

Reply:
I thought about it, but I wasn't sure if it was OK to introduce the `authz_api_base` into `_get_api_session`. If you think that is fine, it would reduce changes to only there instead of four places.

Reply:
sure, i guess this would even allow to omit it in the individual calls (ie just use a `/something` path there)?

Reply:
Exactly!